src/components/subsidy-request-configuration/data/hooks.js

```diff
@@ -18,59 +18,68 @@ export const useSubsidyRequestConfiguration = (enterpriseUUID) => {
+          enterprise_customer_uuid: enterpriseUUID,
+        }),
+      ]);
+      const hasCoupons = couponsData.data.results.length > 0;
+      const hasSubscriptions = subscriptionsData.data.results.length > 0;
 
-        // If the customer has two subsidy types, they are not eligible for Browse & Request in the MVP
-        // A subsidy type of null on the customer configuration indicates that the customer is not eligible
-        let subsidyType = null;
+      // If the customer has two subsidy types, they are not eligible for Browse & Request in the MVP
+      // A subsidy type of null on the customer configuration indicates that the customer is not eligible
+      let subsidyType = null;
 
-        if (!(hasCoupons && hasSubscriptions)) {
-          if (hasCoupons) {
-            subsidyType = SUPPORTED_SUBSIDY_TYPES.coupon;
-          } else if (hasSubscriptions) {
-            subsidyType = SUPPORTED_SUBSIDY_TYPES.license;
-          }
+      if (!(hasCoupons && hasSubscriptions)) {
+        if (hasCoupons) {
+          subsidyType = SUPPORTED_SUBSIDY_TYPES.coupon;
+        } else if (hasSubscriptions) {
+          subsidyType = SUPPORTED_SUBSIDY_TYPES.license;
         }
+      }
 
-        const response = await EnterpriseAccessApiService.createSubsidyRequestConfiguration({
-          enterpriseId: enterpriseUUID,
-          subsidyType,
-        });
+      const response = await EnterpriseAccessApiService.createSubsidyRequestConfiguration({
+        enterpriseId: enterpriseUUID,
+        subsidyType,
+      });
 
-        const customerConfiguration = camelCaseObject(response.data);
+      const customerConfiguration = camelCaseObject(response.data);
+      setSubsidyRequestConfiguration(customerConfiguration);
+    } catch (error) {
+      // log error and do nothing
+      logError(error);
+    }
+  }, [enterpriseUUID]);
 
-        return customerConfiguration;
-      } catch (error) {
-        logError(error);
-        return null;
-      }
-    };
+  const loadSubsidyRequestConfiguration = useCallback(async ({ clearCacheEntry = false }) => {
+    try {
+      const response = await EnterpriseAccessApiService.getSubsidyRequestConfiguration(
+        { enterpriseId: enterpriseUUID, clearCacheEntry },
+      );
+      const customerConfiguration = camelCaseObject(response.data);
+      setSubsidyRequestConfiguration(customerConfiguration);
+    } catch (error) {
+      logError(error);
+      throw error;
+    }
+  }, [enterpriseUUID]);
 
-    const fetchSubsidyRequestConfiguration = async () => {
+  useEffect(() => {
+    if (!enterpriseUUID) {
+      return;
+    }
+
+    const loadConfiguration = async () => {
       try {
-        const response = await EnterpriseAccessApiService.getSubsidyRequestConfiguration(enterpriseUUID);
-        const config = camelCaseObject(response.data);
-        setSubsidyRequestConfiguration(config);
+        await loadSubsidyRequestConfiguration({});
```

Comment:
nit: could the function argument default to `{}` if not provided so, `{}` isn't need here?